examples/swift/headers/header.re

```diff
@@ -0,0 +1,34 @@
+// re2swift $INPUT -o $OUTPUT -i --header lexer/state.swift
+
+/*!header:re2c:on*/
+struct LexerState {
+  var str: UnsafePointer<UInt8>
+  var cur = 0
+  /*!stags:re2c format = "var @@ = -1"; */
+}
+/*!header:re2c:off*/
+
+extension LexerState {
+  mutating func lex() -> Int {
+  let t: Int
+  /*!re2c
+    re2c:header        = "lexer/state.swift";
+    re2c:api           = record;
+    re2c:tags          = 1;
+    re2c:yyfill:enable = 0;
+    re2c:yyrecord      = "self";
+    re2c:YYCURSOR      = "cur";
+    re2c:YYINPUT       = "str";
```

Comment:
Can you rename these two as `yycursor` and `yyinput` and drop configurations? If you do so, please update the autogenerated header (it's easy to forget as the test only check the header pasted at the end of the output file).

Reply:
Done.